samples/nextjs/src/lib/component-props.ts

```diff
@@ -7,14 +7,17 @@ import {
   RouteData,
 } from '@sitecore-jss/sitecore-jss-nextjs';
 
+export type StyleguideSitecoreContextValue = LayoutServiceContext & {
+  route: RouteData;
+};
+
 /**
- * Sitecore Context which you can access using withSitecoreContext
+ * Sitecore Context which you can access using withSitecoreContext/useSitecoreContext
  * @example withSitecoreContext()(ContentBlock)
+ * @example const { sitecoreContext } = useSitecoreContext()
  */
 export type StyleguideSitecoreContext = {
```

Comment:
Since this `StyleguideSitecoreContext` type no longer looks to be used on its own, could this be removed and the `sitecoreContext` just be added directly to `StyleguideComponentWithContextProps`? i.e.
```
export type StyleguideComponentWithContextProps = StyleguideComponentProps & {
  sitecoreContext: SitecoreContextValue;
};
```
 Comments should be moved/updated there as well.

Reply:
@ambrauer This one is used by `StyleguideComponentWithContextProps` and we use it in some class components

Reply:
@ambrauer Agree, did it